server/handlers/destinations.go

```diff
@@ -24,18 +29,20 @@ func DestinationsHandler(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, middleware.ErrResponse("Failed to parse body", err))
 		return
 	}
+
 	err := testDestinationConnection(destinationConfig)
 	if err != nil {
 		c.JSON(http.StatusBadRequest, middleware.ErrResponse(err.Error(), nil))
 		return
 	}
+
 	c.Status(http.StatusOK)
 }
 
 func testDestinationConnection(config *storages.DestinationConfig) error {
```

Comment:
I suppose this func should contain:
1. Creating test eventContext
2. Creating adapter or storage (depend on destination mode - stream or batch)
3. Call test funcs depend on destination mode:
    - stream: create table, insert, delete table
    - batch: TestBatchProcessing()

Reply:
Currently function invokes `TestBatchProcessing` and `TestStreamProcessing` for most types of destinations.
That two functions contain full batch and stream logic for processing events (and logic for cleaning temporary objects).